lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -111,9 +111,12 @@ public function __construct(Configuration $configuration, $version, $class)
         $this->migration = new $class($this);
         $this->version = $version;
 
-        $schemaProvider = new SchemaDiffProvider($this->connection->getSchemaManager(),
-            $this->connection->getDatabasePlatform());
-        $this->schemaProvider = new LazySchemaDiffProvider(new LazyLoadingValueHolderFactory(), $schemaProvider);
+        $this->schemaProvider = $schemaProvider;
+        if(is_null($this->schemaProvider)) {
```

Comment:
Does the new way of setting that property prevent the need for null in the docblock ?